<ui version="4.0" >
 <class>commit</class>
 <widget class="QDialog" name="commit" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>467</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QSplitter" name="splitter" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="MinimumExpanding" hsizetype="MinimumExpanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="handleWidth" >
      <number>2</number>
     </property>
     <widget class="QListWidget" name="commit_list" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="MinimumExpanding" hsizetype="MinimumExpanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize" >
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="font" >
       <font>
        <family>Monospace</family>
       </font>
      </property>
      <property name="alternatingRowColors" >
       <bool>true</bool>
      </property>
      <property name="selectionMode" >
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
     </widget>
     <widget class="QTextEdit" name="commit_text" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="MinimumExpanding" hsizetype="MinimumExpanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize" >
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="font" >
       <font>
        <family>Monospace</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="tabChangesFocus" >
       <bool>true</bool>
      </property>
      <property name="undoRedoEnabled" >
       <bool>false</bool>
      </property>
      <property name="readOnly" >
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Revision Expression:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="revision" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>button_box</tabstop>
  <tabstop>commit_list</tabstop>
  <tabstop>revision</tabstop>
  <tabstop>commit_text</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>commit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>205</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel" >
     <x>205</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>commit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>205</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel" >
     <x>205</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
